src/env.cc

```diff
@@ -231,6 +253,23 @@ void Environment::Start(const std::vector<std::string>& args,
   HandleScope handle_scope(isolate());
   Context::Scope context_scope(context());
 
+  if (*TRACE_EVENT_API_GET_CATEGORY_GROUP_ENABLED(
+      TRACING_CATEGORY_NODE1(environment))) {
+    auto traced_value = tracing::TracedValue::Create();
+    traced_value->BeginArray("args");
+    for (std::string arg : args)
+      traced_value->AppendString(arg);
+    traced_value->EndArray();
+    traced_value->BeginArray("exec_args");
+    for (std::string arg : exec_args)
```

Comment:
`for (const std::string& …)` in both cases, to avoid copies

Reply:
ah, right, doh!

Reply:
@jasnell you fixed the other one :stuck_out_tongue:

Reply:
ha! forgot there were two there :-D